### ISSUE TITLE
Astro v2

### ISSUE THREAD

**Issue by JOELwindows7** [ayoayco/astro-github-stats#5]
# Astro v2 has been released but it seems this integration requires to be updated
[Astro v2 has been released](https://docs.astro.build/en/guides/upgrade-to/v2/#upgrade-astro ). Please edit to add this support.

one of the error when trying to `npm upgrade` & related
```
npm ERR! A complete log of this run can be found in:
npm ERR!     C:\Users\joelr\AppData\Local\npm-cache\_logs\2023-02-05T13_49_09_938Z-debug-0.log
PS W:\starring codes\Astro Projects\perkedel-astro> npm remove astro-embed
npm WARN config global `--global`, `--local` are deprecated. Use `--location=global` instead.
npm WARN ERESOLVE overriding peer dependency
npm WARN While resolving: astro-embed@0.1.3
npm WARN Found: astro@2.0.6
npm WARN node_modules/astro
npm WARN   peer astro@"^1.0.0 || ^2.0.0-beta" from @astro-community/astro-embed-vimeo@0.1.2
npm WARN   node_modules/@astro-community/astro-embed-vimeo
npm WARN     @astro-community/astro-embed-vimeo@"^0.1.0" from astro-embed@0.1.3
npm WARN     node_modules/astro-embed
npm WARN     1 more (@astro-community/astro-embed-integration)
npm WARN   3 more (@astro-community/astro-embed-youtube, ...)
npm WARN
npm WARN Could not resolve dependency:
npm WARN peer astro@"^1.0.0-beta.10" from astro-embed@0.1.3
npm WARN node_modules/astro-embed
npm WARN
npm WARN Conflicting peer dependency: astro@1.9.2
npm WARN node_modules/astro
npm WARN   peer astro@"^1.0.0-beta.10" from astro-embed@0.1.3
npm WARN   node_modules/astro-embed
npm ERR! code ERESOLVE
npm ERR! ERESOLVE could not resolve
npm ERR!
npm ERR! While resolving: astro-github-stats@0.5.0
npm ERR! Found: astro@2.0.6
npm ERR! node_modules/astro
npm ERR!   peer astro@"^1.0.0 || ^2.0.0-beta" from @astro-community/astro-embed-vimeo@0.1.2
npm ERR!   node_modules/@astro-community/astro-embed-vimeo
npm ERR!     @astro-community/astro-embed-vimeo@"^0.1.0" from astro-embed@0.1.3
npm ERR!     node_modules/astro-embed
npm ERR!     @astro-community/astro-embed-vimeo@"^0.1.0" from @astro-community/astro-embed-integration@0.1.2
npm ERR!     node_modules/astro-embed/node_modules/@astro-community/astro-embed-integration
npm ERR!       @astro-community/astro-embed-integration@"^0.1.2" from astro-embed@0.1.3
npm ERR!       node_modules/astro-embed
npm ERR!   peer astro@"^1.0.0 || ^2.0.0-beta" from @astro-community/astro-embed-youtube@0.2.2
npm ERR!   node_modules/@astro-community/astro-embed-youtube
npm ERR!     @astro-community/astro-embed-youtube@"^0.2.0" from astro-embed@0.1.3
npm ERR!     node_modules/astro-embed
npm ERR!     @astro-community/astro-embed-youtube@"^0.2.0" from @astro-community/astro-embed-integration@0.1.2
npm ERR!     node_modules/astro-embed/node_modules/@astro-community/astro-embed-integration
npm ERR!       @astro-community/astro-embed-integration@"^0.1.2" from astro-embed@0.1.3
npm ERR!       node_modules/astro-embed
npm ERR!   2 more (@astrojs/markdown-remark, the root project)
npm ERR!
npm ERR! Could not resolve dependency:
npm ERR! peer astro@"^1.0.0" from astro-github-stats@0.5.0
npm ERR! node_modules/astro-github-stats
npm ERR!   astro-github-stats@"^0.5.0" from the root project
npm ERR!
npm ERR! Conflicting peer dependency: astro@1.9.2
npm ERR! node_modules/astro
npm ERR!   peer astro@"^1.0.0" from astro-github-stats@0.5.0
npm ERR!   node_modules/astro-github-stats
npm ERR!     astro-github-stats@"^0.5.0" from the root project
npm ERR!
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force, or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.
npm ERR!
npm ERR! See C:\Users\joelr\AppData\Local\npm-cache\eresolve-report.txt for a full report.
```
as you can see, I did updated to Astro v2, and some of the plugin including this reported dependency missing & conflict (asking Astro ^v1 package something something)

**Comment by ayo-run** [comment]
@JOELwindows7 Thanks for reporting!

**Comment by ayo-run** [comment]
@JOELwindows7 Released v0.5.2 - you may now update or install

```
npm i astro-github-stats@latest
```
